packages/yarnpkg-core/sources/scriptUtils.ts

```diff
@@ -537,6 +573,7 @@ type ExecutePackageAccessibleBinaryOptions = {
   stdin: Readable | null,
   stdout: Writable,
   stderr: Writable,
+  packageAccessibleBinaries?: PackageAccessibleBinaries,
```

Comment:
I'd flag it as internal, since it's a non-necessary optimization (we could imagine implementing that differently should this part be refactored).

```suggestion
  /** @internal */
  packageAccessibleBinaries?: PackageAccessibleBinaries,
```